webservices/common/models/reports.py

```diff
@@ -49,11 +49,11 @@ class CommitteeReports(PdfMixin, BaseModel):
     refunded_other_political_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_OTHER_POLITICAL_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
     refunded_political_party_committee_contributions_ytd = db.Column(db.Numeric(30, 2), doc=docs.add_ytd(docs.REFUNDED_POLITICAL_PARTY_COMMITTEE_CONTRIBUTIONS))
-    total_contributions_period = db.Column(db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
+    total_contributions_period = db.Column('total_contributions_period', db.Numeric(30, 2), doc=docs.add_period(docs.CONTRIBUTIONS))
```

Comment:
Oh wow, interesting.  It caught my eye because none of the other columns are defined like this.  I'm curious if we're handling other columns that are similar in nature in a different way, and if so how.  I'd have to dig deeper to see, I don't know off the top of my head.